tests/test_edr.py

```diff
@@ -163,10 +172,21 @@ def _assert_progress_range(self, progress, dt, start, stop, step):
             assert ref_line == progress_line
 
 
+def test_edr_to_dict_matches_edr_to_df():
+    array_dict = panedr.edr_to_dict(EDR)
+    ref_df = panedr.edr_to_df(EDR)
+    array_df = pandas.DataFrame.from_dict(array_dict).set_index(
+        "Time", drop=False)
+    assert array_df.equals(ref_df)
+
+
+# TODO: write test that ImportError is raised when pandas not installed
+
```

Comment:
```suggestion
```
This is done now right?

Reply:
I can merge once you clarify this @BFedder

Reply:
Yes indeed, this test is now present. I removed the TODO